README.md

```diff
@@ -26,6 +26,7 @@ Read the documentation and sources of each module how to use it in the project.
 - [logger](./src/logger/README.md)
 - [processing](./src/processing/README.md)
 - [run-in-loop](./src/run-in-loop/README.md)
+- [scripts](./src/scripts/README.md)
```

Comment:
`scripts` sounds too generic. What about `release-scripts`?

Reply:
I figured something generic is actually what we wanted. I'm not attached to it though and can change to `release-scripts`

Reply:
I doubt we'll add more scripts so it seems better to make this more concrete.